strategies/affected/src/test/java/org/arquillian/smart/testing/strategies/affected/ClassDependenciesGraphTest.java

```diff
@@ -141,6 +145,48 @@ public void should_detect_all_changes_transitive() {
     }
 
     @Test
+    public void should_detect_all_changes_adding_package_annotated_transitive() {
+        // given
+        final ClassDependenciesGraph
+            classDependenciesGraph = new ClassDependenciesGraph(new EndingWithTestTestVerifier());
+
+        final String testLocation = ZTest.class.getResource("ZTest.class").getPath();
+        classDependenciesGraph.buildTestDependencyGraph(Arrays.asList(new File(testLocation)));
+
+        // when
+        Set<File> mainObjectsChanged = new HashSet<>();
+        mainObjectsChanged.add(new File(Unwanted.class.getResource("Unwanted.class").getPath()));
```

Comment:
How about extracting this to a simple utility method `asFile("Class.class")`? Would make it a bit easier to follow, as we use it in few places.

Reply:
Or maybe just this part `Unwanted.class.getResource("Unwanted.class").getPath()`. From the test itself it's not really important how we fetch it and it makes it a bit blurry to follow.